content/en/blog/_posts/2025-03-24-kubernetes-1.33-sneak-peek.md

```diff
@@ -0,0 +1,103 @@
+
+
+### DRA’s ResourceClaim Device Status graduates to beta ([KEP-4817](https://kep.k8s.io/4817))
+
+The `Devices` field in `ResourceClaim.Status`, introduced in v1.32, graduates to beta in v1.33. This field allows drivers to report device status data, improving both observability and troubleshooting capabilities.
+
+For example, reporting the interface name, MAC address, and IP addresses of network interfaces in the status of a `ResourceClaim` can significantly help in configuring and managing network services, as well as in debugging network related issues.
+
+You can find more information in [Dynamic Resource Allocation: ResourceClaim Device Status](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/#resourceclaim-device-status).
+
+
+### Ordered namespace deletion ([KEP-5080](https://kep.k8s.io/5080))
+
+This KEP introduces a more structured deletion process for Kubernetes namespaces to ensure secure and deterministic resource removal. The current semi-random deletion order can create security gaps or unintended behaviour, such as Pods persisting after their associated NetworkPolicies are deleted. By enforcing a structured deletion sequence that respects logical and security dependencies, this approach ensures Pods are removed before other resources. The design improves Kubernetes’s security and reliability by mitigating risks associated with non-deterministic deletions.
+
+
+### Enhancements to Kubernetes Job Management and Persistent Volume Policies ([KEP-3850](https://kep.k8s.io/3850), [KEP-3998](https://kep.k8s.io/3998), [KEP-2644](https://kep.k8s.io/2644))
+
+These three KEPs are all graduating to GA to provide better reliability for job and storage handling. [KEP-3850](https://kep.k8s.io/3850) provides per-index backoff limits for indexed jobs, while [KEP-3998](https://kep.k8s.io/3998) extends Job API to define conditions for making an indexed job as successfully completed when not all indexes are succeeded. For storage, [KEP-2644](https://kep.k8s.io/2644) ensures the deletion order of PV-PVC pairs to provide reliable resource cleanup in external storage infrastructure.
+
+## Want to know more?
+
+New features and deprecations are also announced in the Kubernetes release notes. We will formally announce what's new in [Kubernetes v1.33](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.33.md) as part of the CHANGELOG for that release.
+
+You can see the announcements of changes in the release notes for:
+
+* [Kubernetes v1.32](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.32.md)
+
+* [Kubernetes v1.31](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.31.md)
+
+* [Kubernetes v1.30](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.30.md)
+
+* [Kubernetes v1.29](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.29.md)
```

Comment:
Given the [most recent three minor releases](https://kubernetes.io/releases/) are the only ones supported, should I drop v1.29 mention here?

Reply:
This has been removed with db790517fe